ui/v2.5/src/docs/en/ScrapeDevlopment.md

```diff
@@ -76,9 +45,10 @@ The scraping types and their required fields are outlined in the following table
 
 URL-based scraping accepts multiple scrape configurations, and each configuration requires a `url` field. stash iterates through these configurations, attempting to match the entered URL against the `url` fields in the configuration. It executes the first scraping configuration where the entered URL contains the value of the `url` field. 
 
-## Scraper Actions
+    
+## Actions
 
-### Script
+### Python Script
```

Comment:
The `script` action is not python-specific, though it is typically used with python. The action is also named `script`, so it should remain that in the heading.